lib/ship.rb

```diff
@@ -1,7 +1,16 @@
 class Ship 
-attr_reader :name, :length
+attr_reader :name, :length, :health
   def initialize(name, length)
     @name = name
     @length = length
+    @health = 3
```

Comment:
Awesome you got health working. This health return value can change for a submarine of 2 health correct, or is it hard coded 3?